form.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+    function func(String $text, int $size, &$valid, &$errors){
+        $tmp = $_POST["$text"];
+        if(!$tmp){
+            $errors["$text"] = "Полето е задължително."; 
+        } elseif(strlen($tmp) > $size ){
+            $errors["$text"] = "Полето има максимална дължина $size символа.";
+        } else {
+            $valid[$text] = $tmp;
+        }        
+    }
+
+    $valid = array();
+    $errors = array();
+    
+    if( $_POST ){
+        func("title", 150, $valid, $errors);
```

Comment:
Please extract all constant values into constants/variables and give them meaningful names. This way you will know what is the purpose of the "150" value when reading the code without the need to go and check what the function does.

Reply:
This applies for all values in the file.